sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/Disk.cs

```diff
@@ -132,6 +140,7 @@ public Disk(string location, CreationData creationData, string id = default(stri
             OsType = osType;
             HyperVGeneration = hyperVGeneration;
             PurchasePlan = purchasePlan;
+            SupportedCapabilities = supportedCapabilities;
```

Comment:
need to add customization to preserve original constructor signature

Reply:
please check customization folder

Reply:
added customization